docs/contributor-guide/how-to-write-docs.md

```diff
@@ -0,0 +1,32 @@
+# How to write docs
+
+When writing user-facing documentation, pick one of the four options:
+1. Tutorial
+2. How-To guide
+3. Explanation
+4. Reference
+
+For more, see descriptions below or [Divio's documentation system manual](https://documentation.divio.com/).
+
+### Tutorial
+
+Tutorials are learning-oriented and ...
+- must walk-through a self-contained domain like "Deduplicating buildings in Cambridge".
+- must provide any necessary configuration and data for the tutorial.
+
+### How-To guide
+
+How-To guides are problem-oriented and ...
+- must answer a specific, domain-agnostic question like "How to stream datasets out of Tamr"
+
+### Explanation
+
+Explanations are understanding-oriented and ...
```

Comment:
Explanations are understanding-oriented and ...
- Must explain a single concept of the Tamr Python client. If you'd like to write another concept, create it separately.
- Must [keep sentences short](http://www.cs.cmu.edu/afs/cs.cmu.edu/project/cmt-40/kantoo/vol40/doc/kce/styleguide/shortsentences.html).
- May include examples of code or text examples.